docs/buildah-run.md

```diff
@@ -37,9 +37,70 @@ pseudo-TTY to be allocated inside the container connecting the user's "terminal"
 with the stdin and stdout stream of the container.  Setting the `--tty` option to
 `false` will prevent the pseudo-TTY from being allocated.
 
-**--volume, -v** *source*:*destination*:*flags*
-
-Bind mount a location from the host into the container for its lifetime.
+**--volume, -v** *source*:*destination*:*options*
+
+Create a bind mount. If you specify, ` -v /HOST-DIR:/CONTAINER-DIR`, podman
```

Comment:
I'd like to keep up with the paradigm of upper casing Buildah and Podman except when we're giving example commands.

Reply:
Fixed.